rundeckapp/grails-spa/packages/ui/src/components/navbar/main.ts

```diff
@@ -19,4 +24,48 @@ function init() {
       })
 }
 
-window.addEventListener('DOMContentLoaded', init)
\ No newline at end of file
+function initUtil() {
+  const rootStore = getRundeckContext().rootStore
+  const elm = document.getElementById('utilityBar') as HTMLElement
+
+  rootStore.utilityBar.addItems([
+    {
+        "type": "widget",
+        "id": "utility-edition",
+        "container": "root",
+        "group": "left",
+        "class": "rdicon app-logo",
+        "label": "RUNDECK 3.4.0",
+        "visible": true,
+        widget: Vue.extend({
+          components: {RundeckInfoWidget},
+          template: `<RundeckInfoWidget/>`,
+          provide: {
+            rootStore
+          }
+        })
+    },
+    {
+        "type": "action",
+        "id": "utility-support",
+        "container": "root",
+        "group": "right",
+        "class": "fas fa-question-circle fas-xs",
+        "label": "Help",
+        "visible": true,
+        "action": () => {window.open('https://docs.rundeck.com/docs/', "_blank")}
```

Comment:
perhaps the gsp can inject the help url into window.rundeck, and pass that here instead of the hardcoded URL